lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -27,7 +27,7 @@
                     var self = this,
                         deferred = $.Deferred();
                     if (force || this.isStale) {
-                        this.fetch()
+                        this.setPage(1)
```

Comment:
`fetch` isn't overridden by the paginator, so this is just a standard Backbone collection fetch -- grab all the models, and update their data. Call `setPage` instead does eventually run through `fetch`, so the collection is still grabbed from the server. The paginator does override `sync`, so some different code paths are triggered, but for our purposes the only difference is that `setPage` correctly re-sorts the collection. Really, we probably don't want to use `fetch` with anything which inherits from `BackbonePaginator.requestPager` in general.